### ISSUE TITLE
gsync to use for backup entire system to Google Drive?

### ISSUE THREAD

**Issue by orschiro** [iwonbigbro/gsync#82]
Hi @iwonbigbro,

Many thanks for gsync! Would you recommend gsync to create a backup of an entire Linux system to Google Drive?

Can a backup easily be restored on a newly created machine (i.e. is there something like a reverse mode)?

Thanks for your advice!

Robert

**Comment by dmutters** [comment]
I wrote a tutorial to do just that sort of work:

http://wiki.linuxquestions.org/wiki/Rsync_with_Google_Drive

Of course, it would be much wiser, if it will still get the job done, to just backup the user-created data (config files, documents, etc.), rather than doing the whole filesystem, as the latter is likely to take a very long time.

Beware this bug, though:
https://github.com/iwonbigbro/gsync/issues/69
In summary, until a fix is released, uploading large amounts of data will be very problematic, since all gsync users are currently working off of the same maximum data transfer quota.

**Comment by orschiro** [comment]
> I wrote a tutorial to do just that sort of work:

Thanks! I will have a look at it. 

> Of course, it would be much wiser, if it will still get the job done, to just backup the user-created data (config files, documents, etc.), rather than doing the whole filesystem, as the latter is likely to take a very long time.

This seems wise. I will try to move all my personal data to the Google Drive folder first to have them be backed up by the client. The remainder (user configs) will go with symlinks into one folder to be backed up from there. 

Regards,

Robert
